src/Microsoft.Management.Deployment/PackageManager.cpp

```diff
@@ -320,12 +320,14 @@ namespace winrt::Microsoft::Management::Deployment::implementation
     std::shared_ptr<Execution::OrchestratorQueueItem> GetExistingQueueItemForPackage(winrt::Microsoft::Management::Deployment::CatalogPackage package, winrt::Microsoft::Management::Deployment::PackageCatalogInfo catalogInfo)
     {
         std::shared_ptr<Execution::OrchestratorQueueItem> queueItem = nullptr;
+        std::unique_ptr<::AppInstaller::COMContext> context = std::make_unique<::AppInstaller::COMContext>();
         if (catalogInfo)
         {
             // If the caller has passed in the catalog they expect the package to have come from, then only look for an install from that catalog.
             // Fail if they've used a catalog that doesn't have an Id. This can currently happen for Info objects that come from PackageCatalogReference objects for REST catalogs.
             THROW_HR_IF(APPINSTALLER_CLI_ERROR_INVALID_CL_ARGUMENTS, catalogInfo.Id().empty());
-            queueItem = Execution::ContextOrchestrator::Instance().GetQueueItem(Execution::OrchestratorQueueItemFactory::CreateItemForInstall(package.Id().c_str(), catalogInfo.Id().c_str(), nullptr)->GetId().get());
+            auto searchItem = Execution::OrchestratorQueueItemFactory::CreateItemForInstall(std::wstring{ package.Id() }, std::wstring{ catalogInfo.Id() }, std::move(context));

```

Comment:
Now I realize I should have just suggested you make `CreateItemForInstall` take `std::wstring_view`s.  No need to change now...